Source/CarthageKit/Xcode.swift

```diff
@@ -355,6 +357,65 @@ private func mergeExecutables(_ executableURLs: [URL], _ outputURL: URL) -> Sign
 		.then(SignalProducer<(), CarthageError>.empty)
 }
 
+private func findSwiftHeaderFiles(fromExecutableURLs executableURLs: [URL]) -> [URL] {
```

Comment:
What do you think of deleting this function and just mapping over the array using the function below it

Reply:
Deleted this function and used existing extension to find the header file.